challengeMk2/ViewModels/StarSystemsViewModel.cs

```diff
@@ -50,28 +52,28 @@ public StarSystem SelectedSystem
 
         async Task LoadSystemData()
         {
-            InitializeDatabase();
+            await InitializeDatabase();

```

Comment:
A call to initialize the database shouldn't be in a view model ... it should be donne earlier during application startup.

Reply:
Before that version, i made a "loading page" to initialize database during app startup 
(because call it in app.onStart() method doesn't work, first page is always called before onStart).
But Jeremy doesn't like it, because when the first page is loaded, refreshview is already displaying an activity icon and initializing the collection. 
So the Loading page was only to initialize database (too fast to display an activity icon or loading image) and then jump on another "initialize" in the first page.